polaris-e2e/cypress/e2e/api.conversion-failure.cy.ts

```diff
@@ -1,28 +1,45 @@
 /// <reference types="cypress" />
```

Comment:
Changes in the `polaris-e2e/cypress/e2e` folder are to reflect that properties have gone from the document list and now need to be looked up from the normal documents endpoint.